Antenatal supplementation effects on gestational age outcomes (#1213)
ihmeuw/vivarium_research: docs/source/models/intervention_models/maternal_supplementation/index.rst

    - [Keats-et-al-2019-maternal-supplementation]_
  * - BEP
    - MMS
    - Preterm birth, <37 weeks
    - 0.96 (0.80, 1.16); assume no effect
    - [Ota-et-al-2015]_

In order to make these effects compatible with our continuous exposure modeling strategy for LBWSG, we have converted these relative risks of dichotomous outcomes to continuous gestational age "shifts" that result in preterm (and very preterm, if applicable) birth prevalence that replicates the appropriate dichotomous measure of effect. 

The methodology for this conversion was inspired in part by the methodology of the air pollution GBD team in calculating the impact of the risk on LWBSG exposures. As with GBD, we assume that these shifts are independent of any shifts in birth weight. However, rather than implement the conversion using the LBWSG exposure distribution specific to the individual study included in a meta-analysis of the overall effect to find a global shift as GBD did, we used the meta-analyzed global relative risks and applied the conversion for each of our modeled locations, resulting in location-specific continuous shifts that replicate the global dichotomous effect.

> 

Reviewer: ```suggestion
In the case of birth weight shifts, it may be more accurate to assume that the relative risks are generalizable across locations.

```

Assumptions and limitations
~~~~~~~~~~~~~~~~~~~~~~~~~~~

- In the case of IFA, we assume that the entire distribution experiences the same constant shift in gestational age. It is more likely that the lower end of the distribution experiences a greater shift and that the upper end experiences little to no shift (as supported from the MMS evidence). This limitation will result in an underestimation of the impact of the lower end of the distribution (which has a high magnitude of risk, but a low overall exposure) and an overestimation of the impact on the upper end of the distribution (which has lower risk magnitude, but higher overall exposure). However, we have limited data on how to better apply such a shift.

- In the case of MMS, although we have improved the assumption of a single shift applied to the entire distribution through our "dual shift" strategy, it is still limited in that the true shift is likely more of a continuous function with baseline gestational age rather than two conditional values. 

> 

Reviewer: ```suggestion
In particular, a limitation of this approach is the illogical implication that the effect of treatment on a birth that would have been 31.9 weeks without treatment leads to a longer gestation than the effect of the same treatment on a birth that would have been 32.1 weeks without treatment.
```

  - We have some ideas for how we might improve this limitation, including:

    1) Create a function of the dichotomous effect of MMS relative to IFA as a function of baseline gestational age exposure by assuming a linear relationship between the points (32, log(very preterm birth RR)) and (37, log(preterm birth RR)), potentially with some additional constraints (such as constant dichotomous effects above or below some thresholds).

    2) Find gestational age shifts as a function of baseline gestational age that replicates the relative risk values along the line developed in step 1.

  However, we will remain limited by the lack of reported information on the true shape of the association between these "gestational age shifts" and baseline gestational age exposure.

Modeling strategy
~~~~~~~~~~~~~~~~~

> The a supplementation intervention (all regimens) affect infant gestational age at birth exposures, :ref:`which are documented here <2019_risk_exposure_lbwsg>`. 

Reviewer 2: ```suggestion
The supplementation intervention (all regimens) affects infant gestational age at birth exposures, :ref:`which are documented here <2019_risk_exposure_lbwsg>`. 
```
